Library/Formula/youtube-dl-gui.rb

```diff
@@ -0,0 +1,53 @@
+class YoutubeDlGui < Formula
+  desc "A cross platform front-end GUI of the popular youtube-dl."
+  homepage "https://github.com/MrS0m30n3/youtube-dl-gui"
+  url "https://github.com/MrS0m30n3/youtube-dl-gui.git",
+    :revision => "95189d3d82c159e069697fa65058c8a9f88ff686"
```

Comment:
@DomT4 released tarball (https://github.com/MrS0m30n3/youtube-dl-gui/releases/tag/0.3.8) does not contain support for py2app installment. @MrS0m30n3 -> can you please make new release with current py2app which was added in https://github.com/MrS0m30n3/youtube-dl-gui/commit/3d8aa8a0f11442fe96ca41fc9fa2d815488b0afc ?